subsys/mgmt/mcumgr/mgmt/Kconfig

```diff
@@ -25,3 +25,10 @@ config MCUMGR_MGMT_NOTIFICATION_HOOKS
 	  or decline the current operation, by returning false this will signal to the calling
 	  function that the request should be denied, for informal-only notifications or
 	  acceptable, true must be returned by all the registered notification handlers.
+
+config MCUMGR_MGMT_HANDLER_USER_DATA
+	bool "MCUmgr mgmt handler user data support"
+	default n
```

Comment:
`default n` is not needed as that is the default

Reply:
Fixed